app/js/controllers/search.js

```diff
@@ -186,8 +186,15 @@ controllersModule.controller('SearchCtrl', ['$window', '$stateParams', '$rootSco
             SearchService.retrieveMultipleImg(vm.accounts).then(function(data) {
                 vm.accountsPretty = data.users;
                 vm.peopleSearch = true;
-                vm.showMap = false;
+                vm.showMap = false;              
             }, function() {});
+            vm.accountsPretty.forEach(function(ele) {
+                var result = $filter('filter')($rootScope.userTopology.followers, {screen_name : ele.screen_name}, true);
+                if(result.length) {
+                    ele.friend = true;
+                }
+                else ele.friend = false;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'ele'.](https://www.codacy.com/app/sudheesh1995/loklak_webclient/file/2624679912/issues/source?bid=3288768&fileBranchId=3288802#l196)